core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -323,6 +320,7 @@ boolean forceRefresh() {
    * would expire.
    */
   private ListenableFuture<InstanceData> performRefresh() {
+    forcedRenewRateLimiter.acquire(1);
```

Comment:
nit:
```suggestion
    // To avoid unreasonable SQL Admin API usage, use a rate limit to throttle our usage. 
    forcedRenewRateLimiter.acquire(1);
```